src/status.js

```diff
@@ -53,6 +51,53 @@ function addList() {
   form.reset();
 }
 
+function editList(ev) {
+  const buttonId = ev.target.id;
+  const dataGet = localStorage.getItem('todoObject');
+  const data = JSON.parse(dataGet);
+  if (data) {
+    collection = data;
+  }
+  const rtest = collection[collection.findIndex((x) => x.id === parseInt(buttonId, 10))];
+  const last = collection.indexOf(rtest);
+  const test = ev.target.parentNode;
+  const editInput = test.querySelector('p');
+  editInput.contentEditable = true;
+  editInput.classList.add('test');
+  const a = test.querySelector('.fa-ellipsis-v');
+  a.style.display = 'none';
+  const b = test.querySelector('.fa-trash-o');
+  b.style.display = 'block';
+  collection[last].title = editInput.innerHTML;
+
+  editInput.addEventListener('keyup', () => {
+    collection[last].title = editInput.innerHTML;
+    localStorage.setItem('todoObject', JSON.stringify(collection));
+  });
+}
+
+function removeTodo(ev) {
+  const buttonId = ev.target.id;
+  collection = collection.filter(
+    (y) => y !== collection[collection.findIndex(
+      (x) => x.id === parseInt(buttonId, 10),
+    )],
+  );
+  localStorage.setItem('todoObject', JSON.stringify(collection));
+  ShowList(collection);
+}
```

Comment:
We need to update the index once we delete an item.
![image](https://user-images.githubusercontent.com/31897434/127466956-61ef8d1f-7d5b-4be2-866b-d8efeb3011c1.png)

In your case: After I deleted `task 2`, the indexes wasn't updated.
![image](https://user-images.githubusercontent.com/31897434/127467312-042dc3b2-2c73-43e7-827a-0dcbb084ccaf.png)